### ISSUE TITLE
Windows 10 update screen, NVDA moves around while downloading/installing updates

### ISSUE THREAD

**Issue by KevGP** [nvaccess/nvda#12554]
### Steps to reproduce:
Open the Windows update page by either searching for Windows Update in the start menu, or pressing Windows+I. If multiple updates are found and trying to install at once, this is easy to reproduce.
### Actual behavior:
NVDA will jump around and repeat itself, seems the list of updates moves and confuses NVDA. It's extremely dificult to check the status of updates while this is happening.
### Expected behavior:
NVDA should behave normally, as it does with any other list.
#### NVDA installed/portable/running from source:
Install
#### NVDA version:
2020.4
#### Windows version:
Windows 10 21H1 X64 Home
#### Does the issue still occur after restarting your computer?
Yes

**Comment by leonardder** [comment]
@josephsl Any thoughts about this?

**Comment by josephsl** [comment]
Hi, I guess this is due to focus changes caused by updates list refreshing its child elements whenever status changes. I’m thinking this is also the case when using Narrator, so this might be something Microsoft may need to look into after 21H2 era. Thanks.

**Comment by josephsl** [comment]
Hi,

April 2022 update: this may have been fixed in upcoming Windows 11 Version 22H2 (nickel).

Thanks.

**Comment by josephsl** [comment]
Hi,

Reminder: revisit this issue once Windows 10 Version 22H2 is made available to Insiders.

Thanks.

**Comment by Adriani90** [comment]
@josephsl is this issue fixed now?

**Comment by josephsl** [comment]
Hi, sort of resolved in Windows 11 22H2. Thanks.

**Comment by beqabeqa473** [comment]
not fixed in windows 10
